core/templates/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -39,8 +40,8 @@ export interface RteHelperService {
   getRichTextComponents: () => RteComponentSpecs[];
   isInlineComponent: (string) => boolean;
   openCustomizationModal: (
-    customizationArgSpecs, attrsCustomizationArgsDict, onSubmitCallback,
-    onDismissCallback
+    isNewlyCreatedComponent, customizationArgSpecs, attrsCustomizationArgsDict,
```

Comment:
Sorry about this -- I forgot one thing. We usually only start methods/functions with verbs, and we start booleans with non-verbs. If this is a boolean (rather than a function that returns a boolean) could you please use componentIsNewlyCreated instead?

Sorry for missing this earlier. (I'll also withdraw my review and defer to @nikitaevg for the rest.)

Reply:
Done